### PR TITLE
feat: add localization, docker buildx

### LINKED ISSUE #19: add localization & support for building versions for different platforms

### PR DESCRIPTION
**Pull request by chrispinkney**
## Issue
closes #19 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description
This PR adds support for localization. It also allows the CD runner to build images for multiple platforms.

## Testing the PR
n/a

## Checklist

- [ ] **Quality**: This PR builds and passes the respective npm run test and works locally
- [ ] **Tests**: This PR includes thorough tests
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: deployments/Dockerfile [hunk 1]
```diff
@@ -12,14 +12,16 @@ RUN go build -o woodpecker ./cmd/woodpecker
...
+
+ENV TZ=${TZ:-UTC}
 
 WORKDIR /app
 
 COPY --from=builder /app/woodpecker .
 
 LABEL org.opencontainers.image.source="https://github.com/timberware/woodpecker"
-LABEL org.opencontainers.image.description="woodpecker is an app that automatically updates DNS A records for a specified domain and subdomain hosted by Namecheap and/or PorkBun"
+LABEL org.opencontainers.image.description="Woodpecker is a docker-based app that automatically updates DNS A records for specified domain registrars."
```
Comment (manekenpix) [inline deployments/Dockerfile]:
```suggestion
LABEL org.opencontainers.image.description="Woodpecker is a tool that automatically updates DNS A records for specified domain registrars."

```

File: .github/workflows/docker-release.yml [hunk 2]
```diff
@@ -13,17 +13,20 @@ jobs:
...
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build Docker image
-        run: |
-          docker build -t ghcr.io/timberware/woodpecker:latest -f deployments/Dockerfile .
-
-      - name: Push image
+      - name: Build and push multi-platform image
```
Comment (manekenpix) [inline .github/workflows/docker-release.yml]: Is there an action for this?

Comment (chrispinkney) [reply .github/workflows/docker-release.yml]: There's one above, I think? I'm not sure, my docker knowledge is held together by scotch tape:


```yml
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2
```

Comment (manekenpix) [reply .github/workflows/docker-release.yml]: yeah, I mean maybe there's a `build-and-push` action (not instead but besides the one you already have) that supports multiplatform builds.

Comment (chrispinkney) [reply .github/workflows/docker-release.yml]: Added!

**Review by manekenpix — APPROVED** [review]
🤘

**Pull request merged**